bin/node/src/commands/genesis/mod.rs

```diff
@@ -144,45 +133,48 @@ fn create_accounts(
                     auth_scheme,
                 )?;
 
-                AccountData::new(account, Some(account_seed), auth_secret_key)
+                let name = if faucet_count == 0 {
+                    "faucet".to_string()
+                } else {
+                    format!("faucet{faucet_count}")
+                };
```

Comment:
I also don't really like this part of code. :) I wanted to name the file as `faucet.mac` since in the wast majority of cases there will be only one single faucet account in the genesis. But naming it as `faucet0.mac` also may be fine.

Reply:
I've tried to improve the expression here, please take a look. :)